taxi/urls.py

```diff
@@ -16,7 +22,25 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create"
+    ),
+    path(
+        "manufacturers/<int:pk>/update",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update"
+    ),
+    path(
+        "manufacturers/<int:pk>/delete",
```

Comment:
Consider adding a trailing slash to the 'manufacturers/<int:pk>/delete' path for consistency with Django's URL conventions.